apis/securesourcemanager/v1alpha1/types.generated.go

```diff
@@ -47,3 +47,196 @@ type Instance_PrivateConfig struct {
 	//  `projects/{project}/regions/{region}/serviceAttachments/{service_attachment}`.
 	SSHServiceAttachment *string `json:"sshServiceAttachment,omitempty"`
 }
+
+// +kcc:proto=google.cloud.securesourcemanager.v1.Repository
+type Repository struct {
+	// Optional. A unique identifier for a repository. The name should be of the
+	//  format:
+	//  `projects/{project}/locations/{location_id}/repositories/{repository_id}`
+	Name *string `json:"name,omitempty"`
+
+	// Optional. Description of the repository, which cannot exceed 500
+	//  characters.
+	Description *string `json:"description,omitempty"`
+
+	// Optional. The name of the instance in which the repository is hosted,
+	//  formatted as
+	//  `projects/{project_number}/locations/{location_id}/instances/{instance_id}`
+	//  When creating repository via
+	//  securesourcemanager.googleapis.com (Control Plane API), this field is used
+	//  as input. When creating repository via *.sourcemanager.dev (Data Plane
+	//  API), this field is output only.
+	Instance *string `json:"instance,omitempty"`
+
+	// Output only. Unique identifier of the repository.
+	Uid *string `json:"uid,omitempty"`
```

Comment:
We should put these output-only field under SecureSourceManagerRepositoryObservedState

Reply:
Oops I forgot to copy this over from my other branch. Thanks for the catch! Done.